elasticsearch/src/test/java/org/apache/zeppelin/elasticsearch/ElasticsearchInterpreterTest.java

```diff
@@ -94,33 +94,31 @@ public static void populate() throws IOException {
         .setRefresh(true)
         .setSource(jsonBuilder()
           .startObject()
-            .field("date", new Date())
-            .startObject("request")
-              .field("method", METHODS[RandomUtils.nextInt(0, METHODS.length)])
-              .field("url", "/zeppelin/" + UUID.randomUUID().toString())
-              .field("headers", Arrays.asList("Accept: *.*", "Host: apache.org"))
-            .endObject()
-            .field("status", STATUS[RandomUtils.nextInt(0, STATUS.length)])
-            .field("content_length", RandomUtils.nextInt(0, 2000))
-          )
-        .get();
+          .field("date", new Date())
+          .startObject("request")
+          .field("method", METHODS[RandomUtils.nextInt(0, METHODS.length)])
+          .field("url", "/zeppelin/" + UUID.randomUUID().toString())
+          .field("headers", Arrays.asList("Accept: *.*", "Host: apache.org"))
+          .endObject()
+          .field("status", STATUS[RandomUtils.nextInt(0, STATUS.length)])
+          .field("content_length", RandomUtils.nextInt(0, 2000))
+        ).get();
     }
 
     for (int i = 1; i < 3; i++) {
       elsClient.prepareIndex("logs", "http", "very/strange/id#" + i)
         .setRefresh(true)
         .setSource(jsonBuilder()
-            .startObject()
-              .field("date", new Date())
-              .startObject("request")
-                .field("method", METHODS[RandomUtils.nextInt(0, METHODS.length)])
-                .field("url", "/zeppelin/" + UUID.randomUUID().toString())
-                .field("headers", Arrays.asList("Accept: *.*", "Host: apache.org"))
-              .endObject()
-              .field("status", STATUS[RandomUtils.nextInt(0, STATUS.length)])
-              .field("content_length", RandomUtils.nextInt(0, 2000))
-            )
```

Comment:
can we keep the indent? the original one has better readability

Reply:
Thank you for your review. I have adjusted the indentation. How does it look now?